src/main/java/service/impl/FruitReportCreator.java

```diff
@@ -0,0 +1,23 @@
+package service.impl;
+
+import db.Storage;
+import java.util.ArrayList;
+import java.util.List;
+import service.Report;
+import service.ReportCreator;
+
+public class FruitReportCreator implements ReportCreator {
+    public static final String SEPARATOR_NEW_LINE = "\n";
+    public static final String SEPARATOR_COMA = ",";
+    public static final String REPORT_HEADER = "fruit, quantity";
+
+    @Override
+    public Report createReport(Storage fruitStorage) {
```

Comment:
Use the constructor to set the Services, Repositories/DBs
and The methods should basically accept the dtos, models
```suggestion
    public Report createReport() {
```